scripts/sync-and-test-bucket.sh

```diff
@@ -49,6 +49,9 @@ aws_region="$(pulumi -C infrastructure config get 'aws:region')"
 # fail, causing this script to exit nonzero. In either case, it's okay to continue.
 aws s3 mb $destination_bucket_uri --region $aws_region || true
 
+# Tag the bucket with ownership information
```

Comment:
Super-ridiculous complete-sentence nit, sorry -- just something I picked up from others on the team and have come to believe in.  😄
 
```suggestion
# Tag the bucket with ownership information.
```